azure-pipelines.yml

```diff
@@ -761,7 +761,7 @@ jobs:
 
 - job: TestRpmPackagesAzureLinux
   displayName: Test Rpm Package
-  timeoutInMinutes: 120
+  timeoutInMinutes: 180
```

Comment:
Test Rpm Package Azure Linux 3.0 AMD64 fails with
`##[error]The job running on agent pool-ubuntu-2004 38 ran longer than the maximum time of 120 minutes. For more information, see https://go.microsoft.com/fwlink/?linkid=2077134`